Automattic/wp-calypso
# Newsletter settings: re-organise

## Proposed Changes

* Re-organise Newsletter settings page, add new section copies

![image](https://github.com/Automattic/wp-calypso/assets/87168/9318965a-b3ca-444f-b02e-24853a8dd584)

Note that copy is changed in https://github.com/Automattic/wp-calypso/pull/83059 which needs rebasing if this PR merges first.

## Testing Instructions



* Go to Settings → Newsletter and confirm everything works across Simple & Atomic sites.
* You can disable Subscriptions module for Jetpack site from _Discussion settings_. Then look at Newsletter settings page and you should see a banner, and be able to enable module via the banner CTA.

    <img width="1032" alt="Screenshot 2023-10-16 at 20 15 57" src="https://github.com/Automattic/wp-calypso/assets/87168/a4004642-0ad8-410b-b437-ae47d7a0699d">


## Pre-merge Checklist
